crates/red_knot/src/symbols.rs

```diff
@@ -417,7 +485,7 @@ mod tests {
         where
             I: Iterator<Item = SymbolId>,
         {
-            let mut symbols: Vec<_> = it.map(|sym| sym.name.0.as_str()).collect();
+            let mut symbols: Vec<_> = it.map(|sym| sym.name.as_str()).collect();
```

Comment:
Sorry that I removed `itertools`. I'm not a huge fan of it because it hides allocations and most iterators can be implemented by doing a manual collect. However, itertools already is a Ruff dependency and I don't know how the rest of the team feels about it.

Reply:
No worries, avoiding hiding allocations makes sense!